package.json

```diff
@@ -38,7 +38,7 @@
   "devDependencies": {
     "@newrelic/eslint-config": "^0.2.0",
     "@newrelic/newrelic-oss-cli": "^0.1.2",
-    "@newrelic/test-utilities": "^6.5.3",
+    "@newrelic/test-utilities": "^6.5.5",
```

Comment:
Elsewhere we have the test utils being updated to v8.1.0. Do we want to pin this to 6.x?